packages/react/src/evaluation/use-feature-flag.ts

```diff
@@ -13,6 +13,7 @@ import { suspend } from '../common/suspend';
 import { useProviderOptions } from '../provider/context';
 import { useOpenFeatureClient } from '../provider/use-open-feature-client';
 import { FlagQuery } from '../query';
+import { FlagEvaluationOptions } from '@openfeature/web-sdk';
```

Comment:
Please add the import to the existing one.

Reply:
organized imports, thanks.